keyboard.js

```diff
@@ -1704,6 +1705,35 @@ var VKI_attach, VKI_close;
           VKI_mouseEvents(numbkspspan);
           thth.appendChild(numbkspspan);
 
+        if (this.VKI_movement) {
+          function moveAt(pageX, pageY) {
+            this.VKI_keyboard.style.left = pageX + VKI_scrollDist()[0] - keyX + VKI_scrollDist()[0] + 'px';
+            this.VKI_keyboard.style.top = pageY + VKI_scrollDist()[1] - keyY + VKI_scrollDist()[1] + 'px';
+          }
```

Comment:
These references should be `self.` to make explicit we are referencing the parent object.